translationCoreApps/translationWords_Check_plugin
# Add Case Sensitive Checking for some words

#### This pull request addresses:

https://github.com/unfoldingWord-dev/translationCore/issues/1074 highlights some common errors in the Regex. Modifying the regex does not produce the case sensitive checking like required, so a switch case statement is used instead, in order to confirm that capitalized words belong in the right group. 

#### How to test this pull request:

Perform a review of the code, and verify that it does catch the words. Possibly by using console logging to make sure the switch cases are called.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/translationwords_check_plugin/71)
